sensitivity-computation/src/main/java/com/farao_community/farao/sensitivity_computation/SystematicSensitivityAnalysisInterface.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+package com.farao_community.farao.sensitivity_computation;
+
+import com.farao_community.farao.commons.FaraoException;
+import com.farao_community.farao.commons.Unit;
+import com.farao_community.farao.data.crac_api.Crac;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.sensitivity.SensitivityComputationParameters;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+
+/**
+ * A computation engine dedicated to the systematic sensitivity analyses performed
+ * in the scope of the LinearRao.
+ *
+ * @author Viktor Terrier {@literal <viktor.terrier at rte-france.com>}
+ * @author Philippe Edwards {@literal <philippe.edwards at rte-france.com>}
+ * @author Baptiste Seguinot {@literal <baptiste.seguinot at rte-france.com>}
+ */
+public class SystematicSensitivityAnalysisInterface {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SystematicSensitivityAnalysisInterface.class);
+
+    /**
+     * LinearRao configurations, containing the default and fallback configurations
+     * of the sensitivity computation
+     */
+    private SensitivityComputationParameters defaultParameters;
+
+    private SensitivityComputationParameters fallbackParameters;
+
+    /**
+     * The sensitivity provider to be used in the sensitivity computation
+     */
+    private SensitivityProvider sensitivityProvider;
+
+    /**
+     * A boolean indicating whether or not the fallback mode of the sensitivity computation
+     * engine is active.
+     */
+    private boolean fallbackMode = false;
+
+    /**
+     * Constructors
+     */
+    public SystematicSensitivityAnalysisInterface(SensitivityComputationParameters defaultParameters) {
+        this.defaultParameters = defaultParameters;
+    }
+
+    public SystematicSensitivityAnalysisInterface(SensitivityComputationParameters defaultParameters, SensitivityComputationParameters fallbackParameters) {
+        this.defaultParameters = defaultParameters;
+        this.fallbackParameters = fallbackParameters;
+    }
+
+    public SystematicSensitivityAnalysisInterface() {
+        this.defaultParameters = new SensitivityComputationParameters();
+    }
+
+    public void setSensitivityProvider(SensitivityProvider sensitivityProvider) {
+        this.sensitivityProvider = sensitivityProvider;
+    }
```

Comment:
You know what would be amazing ? Something like a builder where you can do .newPtdfSensitivities(GLSKProvider, Set<Cnec>).newRangeActionSensitivities(Set<RangeAction>, Set<Cnec>). This is just a proposition as well :)